### PR TITLE
Alex Noto

### PR DESCRIPTION
**Pull request by noto-alex-j**

### CODE REVIEW

File: lib/reverse_string.rb [hunk 1]
```diff
@@ -1,5 +1,17 @@
 def reverse_characters(str)
+    newarr = []
+	str = str.split("")
+	str.length.times do
+		newarr.push(str.pop)
+		end
```
Comment (halfghaninne) [inline lib/reverse_string.rb:6]: Very nice work! Typically, we indent the line that ends a block the same amount as the beginning line, so line 6 should be in line with line 4.

File: lib/reverse_string.rb [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+	return newarr.join
 end
 
 def reverse_words(str)
+    newarr = []
+	str = str.split()
+	str.length.times do
+		newarr.push(str.pop)
+		end
```
Comment (halfghaninne) [inline lib/reverse_string.rb:15]: Same indentation note.

**Comment by halfghaninne** [comment]
Nice work!

File: lib/reverse_string.rb (hunk 2 above)
Comment (noto-alex-j) [inline lib/reverse_string.rb:15]: Yeah, I have noticed that, for some reason, my line indentations in C9 don't always translate over to Github perfectly. I'll keep an eye out for that!

**Pull request closed without merging**